tutorials/tts/Audio_Codec_Training.ipynb

```diff
@@ -0,0 +1,667 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rtBDkKqVGZJ8"
+      },
+      "source": [
+        "# Introduction"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "pZ2QSsXuGbMe"
+      },
+      "source": [
+        "In this tutorial we show how to run the scripts for training or fine-tuning an audio codecs."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7X-TwhdTGmlc"
+      },
+      "source": [
+        "# License"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fCQUeZRPGnoe"
+      },
+      "source": [
+        "> Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.\n",
+        ">\n",
+        "> Licensed under the Apache License, Version 2.0 (the \"License\"); you may not use this file except in compliance with the License. You may obtain a copy of the License at\n",
+        ">\n",
+        "> http://www.apache.org/licenses/LICENSE-2.0\n",
+        ">\n",
+        "> Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3OZassNG5xff"
+      },
+      "source": [
+        "# Install"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "WZvQvPkIhRi3"
+      },
+      "outputs": [],
+      "source": [
+        "BRANCH = 'main'\n",
+        "# Install NeMo library. If you are running locally (rather than on Google Colab), comment out the below line\n",
+        "# and instead follow the instructions at https://github.com/NVIDIA/NeMo#Installation\n",
+        "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[all]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "tvsgWO_WhV3M"
+      },
+      "outputs": [],
+      "source": [
+        "NEMO_ROOT_DIR = \"/content/nemo\"\n",
```

Comment:
lets change this to current directory path like `./nemo_root` or `./nemo`, otherwise users unnecessarily need to deal with permission issues while creating folder under root `/`